app/resources/en/translation/common.json

```diff
@@ -36,11 +36,12 @@
   "third" : "Third",
   "fourth" : "Fourth",
   "fifth" : "Fifth",
-  "helpTitle" : "Do you need help?",
+  "helpTitle" : "Contact us for help",
+  "helpHeading1": "Telephone",
+  "helpHeading2": "Email",
   "helpText" : "You can call or email us if you&rsquo;re having problems with this service.",
-  "contactTelLabel" : "Phone: {helpLineNumber}",
-  "contactOpeningTimes" : "({openingTimes})",
-  "helpEmailLabel" : "Email:",
+  "contactOpeningTimes" : "{openingTimes}.",
+  "helpEmailLabel" : "Email us at",
   "contactEmailAddress" : "<a href=\"mailto:contactprobate@justice.gov.uk\">contactprobate@justice.gov.uk</a>",
```

Comment:
I know this was already there, but could we not have something like:
```
"contactOpeningTimes" : "{openingTimes}.",
"helpEmailLabel" : "Email us at <a href=\"mailto:{contactEmailAddress}\">{contactEmailAddress}</a>",
```
and get rid of the `contactEmailAddress` field in this file and also add
```
contactEmailAddress: 'contactprobate@justice.gov.uk'
```
to the `links` section in `/app/config.js`?